packages/popover/src/use-popover.ts

```diff
@@ -197,6 +197,9 @@ export function usePopover(props: UsePopoverProps = {}) {
         style: {
           ...props.style,
           transformOrigin: popperCSSVars.transformOrigin.varRef,
+          [popperCSSVars.arrowSize.var]: arrowSize ? px(arrowSize) : undefined,
+          [popperCSSVars.arrowShadowColor.var]: arrowShadowColor,
+          visibility: isOpen ? "visible" : "hidden",
```

Comment:
Any reason to move the `visibility` style?

Reply:
No...that's a mistake. Changing now.